src/rpc/misc.cpp

```diff
@@ -193,7 +233,7 @@ UniValue deriveaddresses(const JSONRPCRequest& request)
     }
 
     FlatSigningProvider provider;
-    auto desc = Parse(desc_str, provider);
+    auto desc = Parse(desc_str, provider, true);
     if (!desc) {
         throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, strprintf("Invalid descriptor"));
```

Comment:
It would be nice to have a dinstinct error for a missing checksum.

Reply:
In general the errors are non-existent. Like, a space causes it to be rejected. Maybe a followup PR to make some common error cases printed?

Reply:
Agree, but I'd prefer to do that in a different change.

Reply:
I think that's fine because both descriptor enhance `importmulti` and `deriveaddress` are new anyway. We can make them more user friendly later.